src/products/mozilla_org/manage_mozillaorg.md

```diff
@@ -0,0 +1,95 @@
+
+The list of completed locales is due to the release team on Friday, 10 days prior to the release date. If the release specific WNP can’t be ready by deadline, the [generic WNP list](https://github.com/mozmeao/www-l10n/blob/master/metadata/firefox/whatsnew/whatsnew.json) will do.
+
+## Staging and Production
+
+Pontoon syncs with the repository every 10 minutes. GitHub makes a production push every 5 minutes. Unless a page is tied to a release on a particular day, chances are, the localization review is done on Production.
+
+When a localized string contains Fluent syntax error, it will be rejected in the  production push, thus the string appears in English.
+
+### Page Activation
+
+It is not unusual when a localized page is mixed with some English strings at a given moment. Here are some scenarios:
+* New page: a localized page will be activated once it reaches 80% completion.
+* Activated page with new string (normal string ID): The strings will appear in English on production right away.
+* Activated page with updated string paired with fallback string ID: The update could be one by one or could be switched on once the tagged strings are fully localized. The developer could control the timing of the update if need be.
+* Activated page with template change: In some cases, a page receives a major update that requires a complete rewrite of the template. If this happens, the old template is kept online only for a defined period of time. When removed, it will cause the URL to redirect users to the English version if the intended localized version is not ready.
+* Activated page missing too many updates: An activated page falling far behind with multiple updates will create bad user experience. We might decide to disable it manually. Removal of a page is done by the Bedrock team.
+
+## Reporting issues
+
+Issues could be reported and questions raised through the following channels:
+
+* [GitHub](https://github.com/mozilla-l10n/www-l10n/pulls)
+* [Bugzilla for l10n](https://bugzilla.mozilla.org/enter_bug.cgi?product=Mozilla+Localizations)
+* [Web dev mailing list](https://groups.google.com/g/mozilla.dev.l10n.web)
+* [Maxtrix Channel](https://chat.mozilla.org/#/room/#l10n-community:mozilla.org)
+* Direct email
+
+Depending on the nature of the issues, here are a few ways to address them:
+
+* Direct to Pontoon for any change request that comes through www-1l0n or Bugzilla. Follow up with the locale manager to review the suggestion.
+* File a bug to [the web team](https://bugzilla.mozilla.org/enter_bug.cgi?product=www.mozilla.org&component=Pages+%26+Content) if a page doesn’t display on production. Usually it impacts more than one locale.
+* File an issue at [Bedrock](https://github.com/mozilla/bedrock/issues) for technical issues or errors in content.
+
+## Distribution of locale coverage
```

Comment:
This is confusing, we normally use "locale coverage" to talk about completion levels. Maybe "Ownership of localization", or something similar? Not that it sound really clear.